src/pages/user/authority/AuthorityCard.tsx

```diff
@@ -13,10 +15,25 @@ const StyledBoxContent = styled.div`
 interface AuthorityCardProps {
   authority: Authority;
 }
+interface Publication {
```

Comment:
eeegentlig burde dette være en type for publication. er dette AlmaPublication eller MarcPublication? eller er det publications lagret i "vårt" arkiv?

Reply:
ja, renamer til `AlmaPublication` og legger til den i `almaApi` også